lib/peatio/securities/key_generator.rb

```diff
@@ -0,0 +1,18 @@
+module Peatio::Securities
+  class KeyGenerator
+
+    def generate!
+      Dir.mkdir('keys') unless File.exists?('keys')
+      OpenSSL::PKey::RSA.generate(2048).yield_self do |p|
+        { public:  Base64.urlsafe_encode64(p.public_key.to_pem), private: Base64.urlsafe_encode64(p.to_pem) }
+          write('keys/rsa-key.prv', p)
```

Comment:
Due to naming convention lets store private key in the file called ```"rsa-key"```
For public we can leave ```rsa-key.pub```